test/e2e/cloud/cluster_upgrade.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package lifecycle
```

Comment:
For the tests we know for sure are gcp specific, can we bundle them together into `gcp.go` file or a `gcp` package - `test/e2e/cloud/gcp/`?

Reply:
i'm +1 on doing that.
a couple of questions would be
- how to name the SIG tag? `sig-cloud-provider-gcp` SGTM
- who to add in the OWNERS files possibly @justinsb and @cheftako

Reply:
Yep, `sig-cloud-provider-gcp` is how we're tagging in test-infra FWIW.

Reply:
ok, i'm going to introduce `//gcp` with the OWNERS and this tag.